src/main/java/com/google/sps/data/User.java

```diff
@@ -2,15 +2,38 @@
 
 import java.util.Collection;
 import java.util.HashSet;
+package profileType;
```

Comment:
Line 1 already has a package command for com.google.sps.data. I don't think we can have two package definitions for the same java file.

Reply:
Ok. done